src/transformime/mathwidget.ts

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+import {
+  Message
+} from 'phosphor-messaging';
+
+import {
+  Widget
+} from 'phosphor-widget';
+
+import {
+  typeset
+} from '../utils/latex';
+
+
+/**
+ * A widget which typesets itself after it is attached.  
+ */
+export
+class MathWidget extends Widget {
```

Comment:
The point is that it typesets its content, whether the content is html or text. It's not just an html renderer. I use it for the the html renderer because we (IIRC) have the assumption that html output is scanned and typeset for math. Hmmm...is that not a valid assumption?